src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,76 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    @Test
+    @DisplayName("쉼표(,)를 구분자로 해서 문자열 \"1,2\"를 split 하면 1, 2를 요소라 갖는 배열이 된다")
+    void splitOneCommaTwo() {
+        String input = "1,2";
+        String[] output = input.split(",");
+        assertThat(output).contains("1", "2");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?

Reply:
이 부분은 제가 잘 이해하지 못한 것 같습니다. 
assertThat 대신에 assertDoesNotThrows 를 사용하라는 의미이실까요?
아니면 assertAll( ) 안에서 actual.forEach(x -> assertThat( ... )) 이런식으로 하라는 의미인가요???